modules/developers-guide/examples/javascript/node-simple-query.js

```diff
@@ -0,0 +1,13 @@
+// For Stargate OSS: SELECT the data to read from the table
+const query = new Query();
+const queryString = 'SELECT firstname, lastname FROM test.users;'
+// Set the CQL statement using the string defined in the last line
+query.setCql(queryString);
+
+// For Stargate OSS and Astra DB: execute the query statement
```

Comment:
I'm a little hesitant to include `and Astra DB` here only because with Astra, you wouldn't pass the `authenticationMetadata` to the `executeQuery` call. (Which we note earlier in the docs)

There's a note at the end of [the authentication section of the Node README](https://github.com/stargate/stargate-grpc-node-client#authentication) that basically says "for the purposes of simplification, all these examples pass metadata, but remember you don't need to do that if you're working with Astra/secure credentials."

I'm sure there's a more artful way to say it, but would it be worth adding something like that to the docs section on authentication? Then we wouldn't have to keep pointing out the difference in every code example.

Reply:
I actually removed the info from the astra-docs version.